src/libs/ReportUtils.ts

```diff
@@ -4540,8 +4540,14 @@ function getIOUSubmittedMessage(reportAction: ReportAction<typeof CONST.REPORT.A
     return Localize.translateLocal('iou.submittedAmount', {formattedAmount: getFormattedAmount(reportAction)});
 }
 
-function getIOUApprovedMessage(reportAction: ReportAction) {
-    return Localize.translateLocal('iou.approvedAmount', {amount: getFormattedAmount(reportAction)});
+function getIOUUnapprovedOrApprovedMessage(reportAction: ReportAction) {
+    let translationKey: TranslationPaths;
+    if (ReportActionsUtils.isApprovedAction(reportAction)) {
+        translationKey = 'iou.approvedAmount';
+    } else {
+        translationKey = 'report.actions.type.unapproved';
```

Comment:
Hmm, Should we add a new translation key 'iou.unapprovedAmount' with the correct translation (need to confirm again). I am not sure if we also want to use 'report.actions.type.unapproved' translation key here

Reply:
because I see in the report detail we use another translation for unapproved action

https://github.com/Expensify/App/blob/6d8e739f7b1c286ae65565a6689b27b5983fe48a/src/languages/es.ts#L1009

Reply:
@iwiznia Could you please help to confirm which translation we should use here?

Reply:
Can you give me a bit more context on what exactly is this showing?

Reply:
When the admin unapprove a request, a system message will display "unapproved X", I am asking the Spanish translation for this system message.

> Hmm, Should we add a new translation key 'iou.unapprovedAmount' with the correct translation (need to confirm again). I am not sure if we also want to use 'report.actions.type.unapproved' translation key here because I see in the report detail we use another translation for unapproved action "unapprove: 'Desaprobar'"


<img width="1512" alt="Screenshot 2024-10-08 at 11 00 57" src="https://github.com/user-attachments/assets/b7eaf83b-466f-4468-b99c-11b1b16f1186">

Reply:
Ah ok thanks! the translation would be `desaprobó`

Reply:
@nkdengineer Please help to update this

Reply:
@DylanDylann updated

Reply:
@nkdengineer I think we should create a new translation key `iou.approvedAmount`